app/(default)/webstories/[slug]/page.tsx

```diff
@@ -32,92 +39,51 @@ const Index: React.FC = () => {
-  };
-
-  const loadCurrentAndNextImage = (index: number) => {
-    if (storyData) {
-      const imageUrls = [
-        storyData[index]?.imageUrl,
-        storyData[index + 1]?.imageUrl,
-      ].filter(Boolean) as string[];
-      preloadImages(imageUrls);
-    }
-  };
-
-  useEffect(() => {
-    loadCurrentAndNextImage(currentStoryIndex);
-  }, [currentStoryIndex, storyData]);
-
   const handleClose = () => {
-    window.close();
+    router.push("/webstories");
   };
 
   const handleStoryIndexChange = (index: number) => {
     setCurrentStoryIndex(index);
   };
 
-  if (isPreloading) {
-    return (
-      <div className="fixed w-full h-full top-0 z-50 flex items-center justify-center">
-        <div className="loaderLoading"></div>
-      </div>
-    );
-  }
-
-  const getImageElement = (url: string) => {
-    const preloadedImage = preloadedImages.find((img) => img.url === url);
-    return preloadedImage ? preloadedImage.img : null;
-  };
-
-  const nextImageIndex = currentStoryIndex + 1;
-
   return (
-    <div className="fixed w-full h-full top-0 z-50 flex items-center justify-center">
-      <div className="absolute w-full h-full top-0 opacity-90 bg-black">
-        <Image
-          src={storyData[0].imageUrl}
-          alt="image"
-          layout="fill"
-          objectFit="cover"
-          className="relative z-10 h-full object-cover w-full blur-2xl opacity-60 md:rounded-xl lg:rounded-xl xl:rounded-xl"
-        />
-      </div>
-      <div className="relative flex flex-col w-full h-full justify-center">
-        {windowWidth > 1024 && (
-          <button
-            onClick={handleClose}
-            className="text-slate-200 border border-solid font-medium bg-secondary-300 p-3 rounded-full shadow-lg absolute top-4 right-4"
-          >
-            <CloseIcon  />
-          </button>
-        )}
-        {storyData ? (
-          <WebStories
-            data={storyData.map((story) => ({
-              ...story,
-              imageElement: getImageElement(story.imageUrl),
-            }))}
-            slug={slug}
-            onStoryIndexChange={handleStoryIndexChange}
+    <RootLayout metadata={metadata} HeaderDisplayed={false}>
+      <div className="fixed w-full h-full top-0 z-50 flex items-center justify-center" data-aos="fade-up">
+        <div className="absolute w-full h-full top-0 opacity-95 bg-black">
+          <Image
+            src={
+              typeof storyData[0].imageUrl === "string"
+                ? storyData[0].imageUrl
+                : storyData[0].imageUrl.src
+            }
+            alt="image"
+            layout="fill"
+            objectFit="cover"
+            className="relative z-10 h-full object-cover w-full blur-2xl opacity-60 md:rounded-xl lg:rounded-xl xl:rounded-xl"
           />
-        ) : (
-          <div>Content Not Available</div>
-        )}
+        </div>
+        <div className="relative flex flex-col w-full h-full justify-center">
+          {windowWidth > 1024 && (
```

Comment:
can we use tailwind here 'hidden' class by default and lg:block to show it only on large screens

Reply:
Added the hidden class.